### ISSUE TITLE
Support `\algnewcommand` from `algorithmicx` when autogenerating  `.cwl` files

### ISSUE THREAD

**Issue by randolf-scholz** [texstudio-org/texstudio#2922]
The popular `algorithmicx` package defines the commands

```latex
\newcommand\algnewcommand{\ALG@defbasecmd\newcommand}%
\newcommand\algrenewcommand{\ALG@defbasecmd\renewcommand}%
```

It would be nice if TexStudio's autogeneration for `cwl` files could treat these like aliases for `\newcommand` and `\renewcommand`, so that user-defined algorithm-commands get recognized.

**Comment by muzimuzhi** [comment]
Cwl for `\algnewcommand` already works for me, in texstudio 4.5.1beta3 on macOS. Related cwl lines were added by https://github.com/texstudio-org/texstudio/commit/2b645f43456f8248590cf27c2b08ca4421a631c7 and then shipped in texstudio v4.0.2. 
![image](https://user-images.githubusercontent.com/6376638/216715785-40a03eaa-9328-4956-8d7c-e058d3334586.png)

**Comment by sunderme** [comment]
this is untested, so please test.
